kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaColumnMappingSuite.scala

```diff
@@ -86,11 +86,32 @@ class DeltaColumnMappingSuite extends DeltaTableWriteSuiteBase with ColumnMappin
       assertColumnMapping(structType.get("a"), 1)
       assertColumnMapping(structType.get("b"), 2)
 
+      val config = getMetadata(engine, tablePath).getConfiguration.asScala
+      assert(config.get(TableConfig.COLUMN_MAPPING_MAX_COLUMN_ID.getKey).contains("2"))
```

Comment:
I didn't realize initially there were `fromMetadata` functions it is probably cleaner to use this then directly checking the configuration map?

Reply:
Yeah makes sense